svo (actions dictionary).xml

```diff
@@ -13693,6 +13502,13 @@ end
 if svo.haveskillset('spirituality') then
   basicdef('heresy', "hunt heresy")
 end
+if svo.haveskillset('zeal') then
+  basicdef('thermalblessing', 'recite fire me', false, nil, true)
+  basicdef('willpowerblessing', 'recite will me', false, nil, true)
+  basicdef('frostblessing', 'recite void me', false, nil, true)
+  basicdef('enduranceblessing', 'recite endurance me', false, nil, true)
+  basicdef('earthblessing', 'recite protection me', false, nil, true)
+end
```

Comment:
Okay, so this function here is overwriting the dictionary entries we created further up, and when it does it removed the prayer balance requirement. That is why it is spamming the prayer balance.